Source/Core/Core/CMakeLists.txt

```diff
@@ -776,4 +776,7 @@ endif()
 if(USE_RETRO_ACHIEVEMENTS)
   target_link_libraries(core PRIVATE rcheevos)
   target_compile_definitions(core PRIVATE -DUSE_RETRO_ACHIEVEMENTS)
-endif()
+	if(RC_CLIENT_SUPPORTS_HASH)
+		target_compile_definitions(core PRIVATE -DRC_CLIENT_SUPPORTS_HASH)
+	endif()
```

Comment:
This uses tabs while the rest of the file uses spaces, please be consistent.

Reply:
This showed up as a conflict multiple times while I was trying to get this merged in and this would explain why. I'll see if I can find a way for VS to make that clearer.